docs/detections/detection-engine-intro.asciidoc

```diff
@@ -51,20 +51,26 @@ To make sure you can access Detections and manage rules, see
 
 [float]
 [[cold-tier-detections]]
-== Compatibility with cold and frozen tier nodes
+== Manage data in cold and frozen tiers
 
-Cold and frozen {ref}/data-tiers.html[data tiers] hold time series data that is only accessed occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold but not frozen tier data for the following {es} indices:
+Cold data tiers store time series data that is accessed infrequently and rarely updated, while frozen data tiers hold time series data that is accessed even less frequently and never updated. If you are automating searches across different {ref}/data-tiers.html[data tiers] using rules, consider the following best practices and limitations.
 
-* Index patterns specified in `securitySolution:defaultIndex`
-* Index patterns specified in the definitions of detection rules, except for indicator match rules 
-* Index patterns specified in the data sources selector on various {security-app} pages
+[float]
+[[best-practices-data-tiers]]
+=== Best practices 
+
+* **Retention in hot tier**: We recommend keeping data in the hot tier for at least 24 hours. {ilm-cap} policies that roll over data more frequently than once every 24 hours can increase the volume of frozen data queried by rules, leading to performance issues.
```

Comment:
```suggestion
* **Retention in hot tier**: We recommend keeping data in the hot tier for at least 24 hours. {ilm-cap} policies that roll over data more frequently than once every 24 hours can increase the volume of frozen data queried by rules, missing out on the performance enhancements offered by hot tier.
```

cc @marshallmain thoughts on the change here? Was trying to make the wording clear that the performance issues rest on the ES side with how frozen works as opposed to something on our end we're doing with frozen data that results in performance issues.

Reply:
Perhaps something like `leading to slower queries and potentially overloading (overwhelming? resource exhaustion?) the frozen tier`? I think calling hot tier a performance enhancement sounds a bit like it's a new feature which could be confusing.